DEPENDENCIES.md

```diff
@@ -6,7 +6,7 @@
 
 | **Component**                 | **Version**                                                                                                                           | **Purpose**                                                                  | **Comments**                                                                                                                                                              |
 |-------------------------------|---------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| OpenShift                     | 4.10                                                                                                                                  | Platform                                                                     | Upgrades to next versions 4.11 need to be tested and approved                                                                                                             |
+| OpenShift                     | 4.11                                                                                                                                  | Platform                                                                     | Upgrades to next versions 4.11 need to be tested and approved                                                                                                             |
```

Comment:
One thought that comes to my mind is, from the user's perspective, does this document sound like a minimum requirement doc? If so, we may need to make it clear. Eg: if we mention 4.11 here, does that translate to - "I need 4.11 cluster. It may not work on 4.10 or below." ?

Would like to hear others' thoughts on this. @Roming22 @adambkaplan @fgiloux

Reply:
>This document lists all the various products/tools used in Pipeline Service along with their current versions.

So it does not sound like a minimum but this is rather the versions we are **currently** using which guarantee that Pipeline Service should work as expected.
If we want to make this file a listing of the minimum requirements then I guess we need to start working on different branches: 4.10, 4.11, etc to be able to provide the right dependencies per openshift version.

Reply:
The version is the current version as per Moemen's comment.

We're currently focusing on a single OCP branch. In the future we might decide to support multiple OCP branches, in which case we'll need to update the CI.